unreleased.md

```diff
@@ -7,3 +7,5 @@
 **Fixed:**
 - react-native-bpk-component-banner-alert:
   - Fixed devDependencies that should be dependencies.
+- bpk-component-modal:
```

Comment:
Changes in the modal also affect dialogs so we should test them and figure out if they are affected / improved.

Also drawers might suffer a similar fate so i reckon we should check them too (outside the scope of this PR).

Reply:
Think dialogs are fine (ie behave the same as before 😄 )

Reply:
And drawers now behave in the same way as the fixed modals, so that's nice